src/parser.mly

```diff
@@ -166,33 +167,39 @@
    strategy. *)
 %token INTENDED_ERROR
 
-(*changes: Conflicts resolution via precedence directives. *)
-
-%left compound_list_newline_list_prec separator_newline_list_prec
-      linebreak_empty_prec
-%left NEWLINE
 
 
 (* -------------------------------------------------------
    The Grammar
    ------------------------------------------------------- *)
-%start<CST.complete_command CST.located>  entry_point
+%start<CST.program CST.located>  entry_point
 %start<unit> intented_error
 %%
 
-entry_point: c=located(complete_command) {
+entry_point: c=located(program) {
   c
 }
 
-complete_command : l=located(clist) s=located(separator) EOF {
-  CompleteCommand_CList_Separator (l, s)
+program : l1=located(linebreak) c=located(complete_commands) l2=located(linebreak) EOF {
+  Program_LineBreak_CompleteCommands_LineBreak (l1, c, l2)
+}
+| l=located(linebreak) {
```

Comment:
There is a new conflict here. Apparently, this reduction can never happen. I couldn't see why